packages/hashes.csv

```diff
@@ -30,16 +30,16 @@ fetchai/connections/tcp,QmS2rmJ9PX2ukS6TqLpUWgRKpt6GTnoHQYnY64XeJb6sDK
 fetchai/connections/webhook,QmSsrWcKhcBoxAWNKQMYPUBr7A3WAkdV62a4D8SMgGasTU
 fetchai/contracts/erc1155,QmbF6t3mjdE6tT7KN3ZhAdn6kxqZ8EXo44mHzEtRTYZ3ia
 fetchai/contracts/scaffold,QmZjRxWo5fJMmVhJFSmdQ3hyR3wMkgBMt1a64RJe8qc6x5
-fetchai/protocols/default,QmU5PttQevBHgignzFSgFHhE8viSsKBPThKxsXGx2mhQXx
-fetchai/protocols/fipa,QmakCsjJAkHiesDsB3r5zW7aMR1LMjBGDQNfHRteoQU5yJ
+fetchai/protocols/default,QmZ76hDzhFWuRnP9TbcNLCpVGxUmXim5cQrtd3PSvQjQjM

```

Comment:
same for default, did not change so you have a local dirty folder

Reply:
Done